README.md

```diff
@@ -61,8 +61,15 @@ Example `config` command:
 # Adapt the path to your kubeconfig file for the garden cluster
 export KUBECONFIG=~/relative/path/to/kubeconfig.yaml
 
-# Fetch cluster-identity of garden cluster
-CLUSTER_IDENTITY=$(kubectl -n kube-system get configmap cluster-identity -ojsonpath={.data.cluster-identity})
+# Method 1 : Fetch cluster-identity of garden cluster from the configmap
+export CLUSTER_IDENTITY=$(kubectl -n kube-system get configmap cluster-identity -ojsonpath={.data.cluster-identity})
```

Comment:
it's not necessary to `export` variables, except for `KUBECONFIG`

Reply:
Good point!